engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
-# Engineering Methodology
-
-Date: 2017-05-09
+# Engineering methodology
+
+Date: 2018-01-15
+
+* [Intro](#intro)
+* [OKRs](#okrs)
+* [The Big Change](#the-big-change)
+* [Kanban](#kanban)
+  * [Company Kanban](#company-kanban)
+  * [Team Kanban](#team-kanban)
+    * [How to pick next issues to worked on?](#how-to-pick-next-issues-to-worked-on)
+* [Leads Meeting](#leads-meeting)
+* [Team Meeting](#team-meeting)
+* [Standup](#standup)
+* [Demo](#demo)
+* [Keeping track of things](#keeping-track-of-things)
+* [Definitions](#definitions)
+  * [What does TODO mean?](#what-does-todo-mean)
+  * [What does Done mean?](#what-does-done-mean)
+  * [What does Blocker mean?](#what-does-blocker-mean)
 
 ## Intro
 
-These are the notes from
- - 2016-09-12 Scrum/OKR
- - 2017-05-08 Sprint #13 Retrospective
+These are the notes from:
+
+* 2017-01-09 Scrum of Scrums
+* 2017-01-10/11 Workflow discussions
+
+Meetings for the Engineering team leads at source{d}.
+It’s the replacement for the [previous workflow](https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md) \w SCRUM
```

Comment:
Error 404.

Reply:
s/\w/with/

Reply:
https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md works for me